requirements.txt

```diff
@@ -4,5 +4,5 @@ scipy
 sympy
 
 # NOTE: switch to non-dev versions when releasing!
-cirq==0.5.0.dev20190416130350
+git+https://github.com/quantumlib/Cirq#egg=cirq
```

Comment:
Let's keep this pinned to a dev version. We can cut a new dev release if necessary.

Reply:
Yea, that was just temporary. Now that cirq-dev is a thing we can pin to that.